src/commons/interfaces.ts

```
@@ -8,3 +8,69 @@ export interface IUserSignUp {
+  validationErrors: string[]
+}
+
+export enum Departamentos {
+  CALEM = 'Centro Acadêmico Lingua Estrangeira Moderna',
+  DAADM = 'Departamento de Administração',
+  DAAGR = 'Departamento de Agronomia',
+  DACOC = 'Departamento de Construção Civil',
+  DACON = 'Departamento de Ciências Contábeis',
+  DAELE = 'Departamento de Engenharia Elétrica',
+  DAFIS = 'Departamento de Física',
+  DAGRO = 'Departamento de Agronomia',
```

ESLint

[error] Duplicate enum member value Departamento de Agronomia.

(@typescript-eslint/no-duplicate-enum-values)